tests/test_pydeseq2.py

```diff
@@ -255,7 +255,7 @@ def test_multifactor_lfc_shrinkage(tol=0.02):
     res = DeseqStats(dds)
     res.summary()
     res.SE = r_res.lfcSE * np.log(2)
-    res.lfc_shrink()
```

Comment:
can you also test that if the wrong coeff is being passed there is an error raised

Reply:
Added in [9b73f0c](https://github.com/owkin/PyDESeq2/pull/101/commits/9b73f0cc1326ff3155eb20035a1574a0125a2b2d)